Documentation/GettingStartedWithMRTKAndXRSDK.md

```diff
@@ -13,27 +13,64 @@ For MRTK 2.3, Windows Mixed Reality is supported on XR SDK.
 ### Required in Unity
 
 1. Go into Unity's Package Manager and install the Windows XR Plugin package, which adds support for Windows Mixed Reality on XR SDK. This will pull down a few dependency packages as well. Ensure the following all successfully installed:
-   1. XR Management
+   1. XR Plugin Management
```

Comment:
```suggestion
   1. XR Plug-in Management
```

Reply:
I (unfortunately) intentionally left these mismatched, as they match the spellings in Unity:

![image](https://user-images.githubusercontent.com/3580640/81008477-db193900-8e07-11ea-90e6-0a25b6b9b9a5.png)

![image](https://user-images.githubusercontent.com/3580640/81008492-e1a7b080-8e07-11ea-914e-8f33a87d12af.png)